aea/connections/p2p/noise/__init__.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# ------------------------------------------------------------------------------
+#
```

Comment:
This cannot be added here, It needs to be under `packages` directory. I will explain you how it works in the call